forsys/fmatrix.py

```diff
@@ -260,24 +256,24 @@ def solve(self, timeseries: Union[str, list] = None, **kwargs) -> dict:
             removed_index = None
         else:
             mprime, b = self.add_mean_one(b)
-            # mprime = self.matrix.T * self.matrix
-            # b = self.matrix.T * b
+            # mprime = self.matrix.T @ self.matrix
+            # b = self.matrix.T @ b

```

Comment:
In principle we could already remove this comments

Reply:
done